src/apps/front-office/home/sections/SectionTopSelling/SectionTopSelling.tsx

```diff
@@ -0,0 +1,124 @@
+import { trans } from "@mongez/localization";
+import HeadName from "apps/front-office/design-system/components/topSelling/HeadName";
+import ProductBox from "apps/front-office/design-system/components/topSelling/ProductBox";
+import {
+  productImg1,
+  productImg10,
+  productImg11,
+  productImg12,
+  productImg2,
+  productImg3,
+  productImg4,
+  productImg5,
+  productImg6,
+  productImg7,
+  productImg8,
+  productImg9,
+} from "./index";
+const SectionTopSelling: React.FC = () => {
+  return (
+    <section className="container my-24 flex gap-6 flex-wrap md:flex-nowrap">
```

Comment:
Split this component into four separate components:
- Top Selling
- Trending Products
- Recently added
- Top Rated